src/app/model/trajectory.ts

```diff
@@ -55,22 +62,30 @@ export class Trajectory implements TrajectoryMeta, TrajectoryData {
   get timestamps() {
     return this.data?.timestamps || []
   }
+  get accuracy() {
+    return this.data?.accuracy || []
+  }
 
   get durationString() {
     return moment.duration(this.durationDays, 'days').humanize()
   }
 
-  get points(): [number, number, Date][] {
-    const points = []
+  get points(): Point[] {
```

Comment:
Yeah. Probably. I just added this as a helper in case some operation benefits from this representation instead of separate arrays. It's not used at the moment, we can probably drop this